src/state/actions/__test__/avgSalaries.test.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import * as rtl from '@testing-library/react';
+import configureStore from 'redux-mock-store';
+import axios from 'axios';
+import MockAdapter from 'axios-mock-adapter';
+import thunk from 'redux-thunk';
+import * as types from '../../types/index';
+
+import { getAvgSalaries } from '../avgSalaries';
+
+const middlewares = [thunk];
+const mockStore = configureStore(middlewares);
+const mock = new MockAdapter(axios);
+
+const avgSalaries = [
+  {
+    interest_id: 2,
+    interest: 'Front End',
+    currency: 'USD',
+    avg: 95000,
+  },
+  {
+    interest_id: 4,
+    interest: 'Full Stack',
+    currency: 'USD',
+    avg: 67500,
+  },
+];
```

Comment:
Did you work on this test yourself? Or was this Colin's work? Because this is exactly the same file I reviewed yesterday on Colin's PR, and I'm certain he's made some recent changes on this.

Reply:
Yes, I wrote the tests for avgSalaries. The mixup arose from the change in how we are writing the test.
You'd recall that we were to work on a single "test-action" branch. I wrote the test before Alex suggested separate branches for each.
Test for companies was a work in progress, so I checked out a new branch off "test-actions", which is why there's duplicates.

Reply:
Okay! That's fine. After pulling from develop can you confirm your `avgSalaries` test passes?